### DISCUSSION TITLE
Gauging interest in a book on LazyVim

### DISCUSSION THREAD

**Discussion by dusty-phillips** [LazyVim/LazyVim#2990] [General]
I’m working on a LazyVim book and I’d like to get a feel for how much interest there is in such a project. The book targets folks who are newish to modal editing and is intended to get you from zero to super productive, with tips and tricks I’ve accumulated over two decades of developing in various vim environments.

The book is looking to be about 20 chapters and over 100k words. A couple questions for anyone interested:

* would you prefer ebook, fancy (colour, hardcover) physical book, trade (black and white, paperback) physical book, or online course/website?
* If this content were free would you support it on patreon?
* Would you back such a project on kickstarter?

About me: I have a history of successful tech books with a couple reputable publishers. This one will be self-published because I am intimately familiar with the process at this point and I want to retain the rights. I have been using vim since the early 2000s, though, like many, I was on the vscode bandwagon for a while (until rsi kicked in and I spent six agonizing months voice coding before going back to modal editing). I have an interest in esoteric programming languages, many of which have been documented on my blog at https://dusty.phillips.codes/ 

The working (and probably production) title is “LazyVim for Ambitious Developers”.

**Comment by surmish** [comment]
Sounds like a great idea tbh. I have been using vim/neovim for a while and would still get benefit from something like this.

**Reply by dusty-phillips** [reply]
Preview is here! https://lazyvim-ambitious-devs.phillips.codes/

**Comment by the-mrd** [comment]
Thank you very much for your effort. Answers below.

> I’m working on a LazyVim book and I’d like to get a feel for how much interest there is in such a project. The book targets folks who are newish to modal editing and is intended to get you from zero to super productive, with tips and tricks I’ve accumulated over two decades of developing in various vim environments.

I am interested both for myself, a longtime vim user, but also for younger colleagues who keep asking me for neovim books. I keep recommending the classic vi/vim books, but those lack discussions on treesitter, LSP and other "modern" neovim features. 
  
>     * would you prefer ebook, fancy (colour, hardcover) physical book, trade (black and white, paperback) physical book, or online course/website?
Ebook with nice colors.

>     * If this content were free would you support it on patreon?
Yes, either patreon or github sponsorship.
 
>     * Would you back such a project on kickstarter?
Yes again. 
 
> The working (and probably production) title is “LazyVim for Ambitious Developers”.

Good title, but it feels like there should be something catchier with lazyVim
